assets/scss/projects.scss

```diff
@@ -2,6 +2,7 @@
 @import "images";
 $light-green: #26A197;
 $pink: #ff1493;
+$hint-colour: #696F73;
```

Comment:
Is this change needed?

Reply:
The hint colour is used at the bottom of the css file but never defined so I was getting an error when I built it in projects, so this fixes that. Sorry should have also mentioned this in the PR